openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1241,7 +1241,7 @@ Authorization: Bearer czZCaGRSa3F0MzpnWDFmQmF0M2JW
 {
   "notification_id": "3fwe98js",
   "event": "credential_deleted",
-  "error_description": "..."
+  "event_description": "User pressed 'cancel' button"
```

Comment:
suggestion was to use “credential_faliure” event and put a description “could not retrieve issuer key”.